packages/amplify-cli/src/extensions/amplify-helpers/remove-resource.ts

```diff
@@ -40,13 +40,15 @@ export async function removeResource(
 ) {
   const amplifyMeta = stateManager.getMeta();
 
-  if (!amplifyMeta[category] || Object.keys(amplifyMeta[category]).length === 0) {
+  if (!amplifyMeta[category] || Object.keys(amplifyMeta[category]).filter(r => !!amplifyMeta[category][r].providerPlugin).length === 0) {
```

Comment:
this is out of scope for this PR as there were no specific requirements around it, since it could break existing deployments I considered the imported resources as immutable ones.